package.json

```diff
@@ -7,9 +7,11 @@
     "lint:fix": "eslint --fix src tests",
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "ios:dev": "react-native run-ios --scheme ReactNativeBase-Develop",
+    "ios:qa": "react-native run-ios --scheme ReactNativeBase-QA",
     "ios:staging": "react-native run-ios --scheme ReactNativeBase-Staging",
     "ios:prod": "react-native run-ios --scheme ReactNativeBase-Prod",
     "android:dev": "react-native run-android --variant=devDebug --appIdSuffix=dev",
+    "android:qa": "react-native run-android --variant=qaDebug --appIdSuffix=qa",
```

Comment:
Perhaps add also `"android:release:qa"` and `android:bundle:qa`?

Reply:
good catch, i didn't saw it due to docs in Notion only use the the run command